android/app/src/main/java/org/fossasia/openevent/utils/CompressImage.java

```diff
@@ -0,0 +1,30 @@
+package org.fossasia.openevent.utils;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * Created by free4murad on 3/9/17.
+ */
+
+public class CompressImage {
+    public static String compressImage(String file) {
+        URL url;
+        String temp = "";
+        int indexOfPath = -1;
+        try {
+            url = new URL(file);
+        } catch (MalformedURLException e) {
+            return file;
+        }
+        indexOfPath = file.indexOf(url.getPath());
+        if(!url.getHost().endsWith(".rsz.io")) {
+            temp = file.substring(0, indexOfPath) + ".rsz.io" + file.substring(indexOfPath) + "?width=300";
+            file = temp;
+        }
+        if(url.getProtocol().equals("https")){
+            file = temp.replace("https", "http");
```

Comment:
Can you please add the `https` & `html` in strings.xml instead of hardcoding. :+1: 
Also `.rsz.io` :)